django/sierra/sierra/routers.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.db import DatabaseError
+
+
+class SierraRouter(object):
+    """
+    The DB router used by the Catalog API project.
+    """
+    def db_for_read(self, model, **hints):
```

Comment:
We should put a blank line after the docstring so the method is _surrounded_ by single blank lines, per [PEP-8](https://www.python.org/dev/peps/pep-0008/#blank-lines)